woocommerce/woocommerce-android
# [Mobile Payments] Record country on the selection payment method screen

Closes: #9615 


### Description

We noticed that many events related to the payments have `"country":"unknown"` property. This PR should fix that

### Testing instructions

To reproduce:
* Either have a clean install or use flipper to reset `CARD_READER_LAST_SUCCESSFUL_PAYMENT_TIME` preference to 0
* Restart the app
* Create an order
* Collect payment
* Notice in logcat
```
card_present_tap_to_pay_not_available, Properties: {"reason":"NfcNotAvailable","source":"payment_methods","plugin_slug":"woocommerce-payments","country":"unknown","blog_id":192152755,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true}
```

Do the same on this branch and notice that the same event has a country set.
